tests/mpi_test.c

```diff
@@ -6,51 +6,51 @@
 static int s_radix_to_bin_test(void)
 {
    /* RADIX 16 */
-   const char *ghex = "2";
-   const char *phex = "FFFFFFFFFFFFFFFFC90FDAA22168C234C4C6628B80DC1CD129024E088A67CC74020BBEA63B139B22"
+   const char ghex[] = "2";
+   const char phex[] = "FFFFFFFFFFFFFFFFC90FDAA22168C234C4C6628B80DC1CD129024E088A67CC74020BBEA63B139B22"
```

Comment:
nice. I also prefer `buf[]` rather than `buf*` since then you can use `sizeof(buf)-1` for the buffer length.

Reply:
you meant the string length? :)

btw. one can also use `strlen()` for const strings, this is usually evaluated at compile time and not causing a call to the function